<?xml version="1.0" encoding="UTF-8" ?>

<launch>
    <include file="$(find rover_driver_base)/launch/rover.launch"/> 
    
    <node name="task_server" pkg="floor_nav" type="task_server" output="screen" >
        <!-- After install -->
        <!-- <param name="lib_path" value="$(find floor_nav)/tasks"/> -->
        <!-- While developing -->
        <param name="lib_path" value="$(find floor_nav)/../../devel/share/floor_nav/tasks"/>
        <param name="base_frame" value="rover/body"/>
        <param name="reference_frame" value="rover/ground"/>
        <remap from="/mux/autoCommand" to="/rover/twistCommand"/>
        <!--<param name="base_frame" value="body"/>
        <param name="reference_frame" value="map"/>-->
    </node>
    <!-- For the rosControlKinect scene
    <node name="viz_graph" pkg="floor_graph" type="viz_graph.py" output="screen">
    </node>
    -->

</launch>
